modules/shared/src/main/scala/io/constellationnetwork/schema/swap.scala

```diff
@@ -158,14 +159,35 @@ object swap {
   @newtype
   case class SpendTransactionFee(value: PosLong)
 
+  @derive(decoder, encoder, order, ordering, show)
+  sealed trait SpendTransaction extends SwapTransaction
```

Comment:
I think it shouldn't extend the SwapTransaction

Reply:
👍🏼